Letterbook.Core.Tests/DeliveryWorkerTests.cs

```diff
@@ -24,33 +27,35 @@ public class DeliveryWorkerTests : WithMocks, IClassFixture<JsonLdSerializerFixt
     private readonly CloudEvent _event;
     private readonly DeliveryObserver _observer;
 
-    public DeliveryWorkerTests(ITestOutputHelper output, JsonLdSerializerFixture serializer)
+    public DeliveryWorkerTests(ITestOutputHelper output, JsonLdSerializerFixture serializer,
+        TestOutputLoggerFixture outputLogger)
     {
         _output = output;
+        _outputLogger = outputLogger;
         _output.WriteLine($"Bogus seed: {Init.WithSeed()}");
-        _mockWorker = new Mock<DeliveryWorker>(Mock.Of<ILogger<DeliveryWorker>>(), AccountProfileMock.Object,
-            ActivityPubClientMock.Object);
+
+        _mockWorker = new Mock<DeliveryWorker>(outputLogger.CreateLogger<DeliveryWorker>(output),
```

Comment:
Actually, I think this is really useful to highlight.

OutputLogger creates an ILogger<T> that can be injected into systems under test, and which is backed by the XUnit ITestOutpuHelper. This makes it straightforward in most cases to get the service's logs in the test output. Possibly very helpful in debugging test failures. Especially in message workers like this one.